package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "docs",
+  "name": "qiskit-documentation",
```

Comment:
This is the name people will use in `dependencies` in their `package.json`, so it should not be so vague.